src/pages/settings/Profile/PersonalDetails/AddressPage.js

```diff
@@ -109,12 +115,25 @@ class AddressPage extends Component {
 
         // Add "Field required" errors if any required field is empty
         _.each(requiredFields, (fieldKey) => {
-            if (!_.isEmpty(values[fieldKey])) {
+            if (ValidationUtils.isRequiredFulfilled(values[fieldKey])) {
                 return;
             }
             errors[fieldKey] = this.props.translate('common.error.fieldRequired');
         });
 
+        // If no country is selected, default value is "-"
+        const countryDetails = lodashGet(CONST.COUNTRY_ZIP_VALIDATION_REGEX, values.country, {});
```

Comment:
Can we make this variable name clearer? Maybe like this?

```suggestion
        const countryRegexDetails = lodashGet(CONST.COUNTRY_ZIP_VALIDATION_REGEX, values.country, {});
```